reference/conanfile/methods.rst

```diff
@@ -1508,6 +1508,14 @@ self.folders
   method in a ``conanfile.py``. In the cache, when running the :command:`conan create`, this subfolder will be relative to the root
   build folder and when running the :command:`conan imports` command it will be relative to the current working directory.
 
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
```

Comment:
I think we could start adding the exact tag to each one:

```suggestion
Available since: `1.46.0 <https://github.com/conan-io/conan/releases/tag/1.46.0>`_
```